invisible_cities/reco/icaro_components_test.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+
+import numpy          as np
+import numpy.testing  as npt
+import pandas         as pd
+import scipy.optimize as so
+
+from hypothesis             import given, settings
+from hypothesis.strategies  import floats, integers
+
+from .. reco                import icaro_components as icarcomp
+from ..types.symbols        import KrFitFunction
+from .. evm.ic_containers   import FitFunction
+from .. core.fit_functions  import expo
+
+
+@given(floats(min_value= 0, max_value= 10),
+       floats(min_value=10, max_value= 20),
+       floats(min_value= 1, max_value=100),
+       floats(min_value= 0, max_value= 10))
+def test_lin_function_output_values(x_min, x_max, a, b):
+    x = np.array([x_min, x_max])
+    y = a + b * x
+    a_test, b_test = icarcomp.lin_seed(x, y)
+    assert np.isclose(a_test, a)
```

Comment:
A blank line between definitions and assertions is good. Also applies to the next test